crates/bitwarden-vault/src/cipher/cipher_client/admin/create.rs

```diff
@@ -0,0 +1,189 @@
+use bitwarden_api_api::models::CipherCreateRequestModel;
+use bitwarden_core::{
+    ApiError, MissingFieldError, NotAuthenticatedError, UserId, key_management::KeyIds,
+};
+use bitwarden_crypto::{CryptoError, IdentifyKey, KeyStore};
+use bitwarden_error::bitwarden_error;
+use thiserror::Error;
+#[cfg(feature = "wasm")]
+use wasm_bindgen::prelude::*;
+
+use crate::{
+    Cipher, CipherView, VaultParseError,
+    cipher::cipher::PartialCipher,
+    cipher_client::{
+        admin::CipherAdminClient,
+        create::{CipherCreateRequest, CipherCreateRequestInternal, CreateCipherError},
+    },
+};
+
+#[allow(missing_docs)]
+#[bitwarden_error(flat)]
+#[derive(Debug, Error)]
+pub enum CreateCipherAdminError {
+    #[error(transparent)]
+    Crypto(#[from] CryptoError),
+    #[error(transparent)]
+    Api(#[from] ApiError),
+    #[error(transparent)]
+    VaultParse(#[from] VaultParseError),
+    #[error(transparent)]
+    MissingField(#[from] MissingFieldError),
+    #[error(transparent)]
+    NotAuthenticated(#[from] NotAuthenticatedError),
+}
+
+impl<T> From<bitwarden_api_api::apis::Error<T>> for CreateCipherAdminError {
+    fn from(val: bitwarden_api_api::apis::Error<T>) -> Self {
+        Self::Api(val.into())
+    }
+}
+
+/// Wraps the API call to create a cipher using the admin endpoint, for easier testing.
+async fn create_cipher(
+    request: CipherCreateRequestInternal,
+    encrypted_for: UserId,
+    api_client: &bitwarden_api_api::apis::ApiClient,
+    key_store: &KeyStore<KeyIds>,
+) -> Result<CipherView, CreateCipherError> {
```

Comment:
⛏️ Should this be `CreateCipherAdminError` instead of `CreateCipherError`?
```suggestion
) -> Result<CipherView, CreateCipherAdminError> {
```